### ISSUE TITLE
Still working on this?

### ISSUE THREAD

**Issue by majortom64** [pigeon-cms/pigeon#36]
Curious if you are still working on this?Seems like a nice idea, but it does not seem you are still actively working on it.

**Comment by nonfungibletunji** [comment]
I have time, to work on this, should we just fork. @majortom64

**Comment by majortom64** [comment]
@stunjiturner I think forking, but working based on the project plan and passing things back as PR's makes the most sense. Updating to Vapor 4 seems pretty important.

**Comment by nonfungibletunji** [comment]
I agree, but would like something from @hallee by way of communication. About his future plans.

**Comment by majortom64** [comment]
That makes sense. Could reach out with an email or Twitter DM to him and see what he says. :-)

**Comment by hallee** [comment]
Hi @stunjiturner @majortom64,

Other projects have been taking up my time and I haven't looked at this project in awhile.

I plan to move to Vapor 4 and now that that's released, I might take a look at this project again in the coming months. I'm also waiting on https://github.com/GraphQLSwift/GraphQL/pull/57 to get merged.

I'm open to contributions if you'd like to help!

**Comment by nonfungibletunji** [comment]
@hallee Can I use this code. Fork and put under another repo? So it can be updated and worked on...
